LICENSE

```diff
@@ -1,6 +1,6 @@
-MIT License
+The MIT License (MIT)
 
-Copyright (c) 2019 MOXY
+Copyright (c) 2019 Made With MOXY Lda <hello@moxy.studio>
```

Comment:
Should we use `2019` or `2017` like [js-proper-url-join](https://github.com/moxystudio/js-proper-url-join/blob/master/LICENSE)?

Reply:
2019